config-overrides.js

```diff
@@ -1,9 +1,17 @@
-module.exports = function override(webpackConfig) {
-  webpackConfig.module.rules.push({
-    test: /\.mjs$/,
-    include: /node_modules/,
-    type: "javascript/auto",
-  });
+module.exports = {
+  webpack: function override(webpackConfig) {
+    webpackConfig.module.rules.push({
+      test: /\.mjs$/,
+      include: /node_modules/,
+      type: "javascript/auto",
+    })
 
-  return webpackConfig;
-};
+    return webpackConfig
+  },
+  devServer: function (configDevServer) {
+    return function (proxy, allowedHosts) {
+      const config = configDevServer(proxy, allowedHosts)
+      return { ...config, public: "alberto.local" }
+    }
+  },
```

Comment:
This bit allows Webpack to serve Alberto locally at a specific domain (e.g. `https://alberto.local:3000`) provided you also add the following line to your `/etc/hosts`:

```
127.0.0.1  alberto.local
127.0.0.1  wallet.end-labs.local
```

It's useful in local development to test whether the hostname detection is working.